app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditViewModel.kt

```diff
@@ -14,18 +13,22 @@ import org.oppia.android.util.data.DataProviders.Companion.toLiveData
 import org.oppia.android.util.logging.ConsoleLogger
 import javax.inject.Inject
 
-// TODO(#1633): Fix ViewModel to not depend on View
 /** The ViewModel for [ProfileEditActivity]. */
 @ActivityScope
 class ProfileEditViewModel @Inject constructor(
-  private val activity: AppCompatActivity,
   private val logger: ConsoleLogger,
   private val profileManagementController: ProfileManagementController
 ) : ObservableViewModel() {
   private lateinit var profileId: ProfileId
-  private lateinit var switch: Switch
 
-  lateinit var profileName: String
+  private val _isAllowedDownloadAccess = MutableLiveData<Boolean>()
+
+  /**
+   * This variable is used to expose the switch state
+   * that determines whether the access to download is allowed or not
+   * to the presenter
+   **/
```

Comment:
```suggestion
  /** Specifies whether download access has been enabled by the user. */
```

Couple things to note:
1. KDoc ends with ``*/`` not ``**/``
2. KDocs should end with punctuation
3. KDoc lines should be wrapped (not applicable in my suggestion since I simplified it)
4. Field documentation should describe _what_ the value represents, not who uses it and usually not how it's assigned (though in some cases this is important)